docs/_sources/calling_python.rst.txt

```diff
@@ -1,13 +1,15 @@
 Remote Calling Python from Erlang
```

Comment:
This file is same or almost same as `calling_python.rst` what is happening?

Reply:
everything under docs is generated by sphinx, I'm guessing it places the source files there